docs/how-tos/nexus-validation-dev-notes.md

```diff
@@ -0,0 +1,473 @@
+# NeXus Validation Dev. Notes
+
+# 1\. Overview of validation methods
+
+nexusformat.org listed two programs for the verification and validataion of NeXus files:  
+[https://manual.nexusformat.org/validation.htm](<https://manual.nexusformat.org/validation.html>)
+
+1. nxvalidate
+
+2. punx
+
+## nxvalidate
+
+[https://github.com/nexusformat/cnxvalidate](<https://github.com/nexusformat/cnxvalidate>)
+
+### Pro:
+
+1. FAIRmat and NIAC NeXus definiton can be used.
+
+2. Easy to install with Linux
+
+3. Points out some Errors with pynxtools-ellips created file - our fault?
+
+### Con: 
+
+1. Is C/C++ based
+
+2. Not possible for Windows (cmake did not found the libxml2 library. Other libs were found after some work)
+
+3. Last update 2 years ago - Repo is dead?
+
+4. Unclear if every error is 100% a fault on our side - Florian was unsure/had some concerns about that.
```

Comment:
If this is supposed to be part of the actual official docs, these comments must be removed.

Reply:
No, this is intended to be internal. Is there something to locate this? I just did not list it in the mkdocs.yaml to "hide" it (yea, i know...).